content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -37,7 +47,65 @@ In the `generator` block of your `schema.prisma` file, add the following:
 binaryTargets = ["native", "linux-arm64-openssl-1.0.x"]
 ```
 
-## Loading environment variables via a <inlinecode>.env</inlinecode> file
+### Prisma CLI binary targets
+
+While we do not recommend running migrations in an AWS Lambda context, some applications will require it. In these cases, you can use the [PRISMA_CLI_BINARY_TARGETS](https://www.prisma.io/docs/reference/api-reference/environment-variables-reference#prisma_cli_binary_targets) environment variable to make sure that `prisma migrate` has access to the correct schema engine.
+
+In the case of AWS lambda, you will want to have the following line in your `.env` file
```

Comment:
Am I not allowed to just put this into a normal envirvonment variable instead of the `.env` file that I am maybe not using?

Reply:
```suggestion
In the case of AWS lambda, you will want to have the following environment variable:
```